GOVERNANCE.md

```diff
@@ -0,0 +1,179 @@
+satisfying the needs of those users. Common user contributions include (but are
+not limited to):
+
+- evangelising about the project (e.g. a link on a website and word-of-mouth
+awareness raising)
+- informing developers of strengths and weaknesses from a new user perspective
+- providing moral support (a ‘thank you’ goes a long way)
+
+Users who continue to engage with the project and its community will often
+become more and more involved. Such users may find themselves becoming
+contributors, as described in the next section.
+
+## Contributors
+
+Contributors are community members who contribute in concrete ways to the
+project. Anyone can become a contributor, and contributions can take many
+forms. There is no expectation of commitment to the project, no specific skill
+requirements and no selection process.
+
+In addition to their actions as users, contributors may also find themselves
+doing one or more of the following:
+
+- supporting new users (existing users are often the best people to support
+new users)
+- reporting bugs
+- identifying requirements
+- providing graphics and web design
+- programming
+- assisting with project infrastructure
+- writing documentation
+- fixing bugs
+- adding features
+
+Contributors engage with the project through the issue tracker and mailing
+list, or by writing or editing documentation. They submit changes to the
+project itself via patches, which will be considered for inclusion in the
+project by existing maintainers (see next section). The developer mailing list
+is the most appropriate place to ask for help when making that first
+contribution.
+
+As contributors gain experience and familiarity with the project, their profile
+within, and commitment to, the community will increase.
+A contributer who shows an above-average level of contribution to the project,
+particularly with respect to its strategic direction and long-term health, may
+be nominated to become a maintainer. This role is described below.
+
+## Maintainers 
+
+Maintainers are community members who have shown that they are committed to the
+continued development of the project through ongoing engagement with the
+community.
+
+Typically, a potential maintainer will need to show that they have an
+understanding of the project, its objectives and its strategy. They will also
+have provided valuable contributions to the project over a period of time.
+
+The project maintainers consists of those individuals identified as
+‘project owners’ on the development site. The maintainers have additional
+responsibilities over and above those of a contributor. These responsibilities
+ensure the smooth running of the project. Maintainers are expected to review
+code contributions, participate in strategic planning, approve changes to the
+governance model and manage the copyrights within the project outputs.
+
+Maintainers do not have significant authority over other members of the
```

Comment:
There is no such list, is there?

Reply:
It is an abstract one :)
The names of the maintainers are specified in a file, so we have a list of mails. Would you like to suggest an alternative phrasing?

Reply:
I rephrased.